cli/lsp/config.rs

```diff
@@ -1071,6 +1076,10 @@ impl Config {
     paths
   }
 
+  pub fn persist_log(&self) -> bool {
+    self.settings.unscoped.persist_log
```

Comment:
Bike shedding: What do you think about `"logFile": true`? It would allow more easily for a string value like `"logFile": "./some_path.log"` in the future?